python-avd/pyavd/_eos_cli_config_gen/j2templates/documentation/ethernet-interfaces.j2

```diff
@@ -34,14 +34,14 @@
 {%                 set mode = port_channel_interface.switchport.mode | arista.avd.default('-') %}
 {%                 if port_channel_interface.switchport.access_vlan is arista.avd.defined or
                     port_channel_interface.switchport.trunk.allowed_vlan is arista.avd.defined %}
-{%                     set vlans = [] %}
+{%                     set switchport_vlans = [] %}
```

Comment:
changing the variable name in templates, as it could refer to `vlans` data-model and generate unexpected data here.